src/NUnitEngine/nunit.engine.core/RuntimeFramework.cs

```diff
@@ -641,6 +684,73 @@ private static void AddMonoFramework(Version frameworkVersion, string profile)
 
             _availableFrameworks.Add(framework);
         }
+
+        private static void FindDotNetCoreFrameworks()
+        {
+            const string WINDOWS_INSTALL_DIR = "C:\\Program Files\\dotnet\\";
+            const string LINUX_INSTALL_DIR = "/usr/shared/dotnet/";
+            string INSTALL_DIR = Path.DirectorySeparatorChar == '\\'
+                ? WINDOWS_INSTALL_DIR
+                : LINUX_INSTALL_DIR;
+
+            if (!Directory.Exists(INSTALL_DIR))
+                return;
+            if (!File.Exists(Path.Combine(INSTALL_DIR, "dotnet.exe")))
+                return;
+
+            string runtimeDir = Path.Combine(INSTALL_DIR, "shared\\Microsoft.NETCore.App");
```

Comment:
Niggle - should this be a forward slash for Linux? Not sure if Path.Combine corrects this for us?

Reply:
I think you're right.